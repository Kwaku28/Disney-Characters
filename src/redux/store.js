import { configureStore } from '@reduxjs/toolkit';
import charactersReducer from './characters/charactersSlice';
import profileReducer from './profile/profileSlice';

const store = configureStore({
  reducer: {
    characters: charactersReducer,
    profile: profileReducer,
  },
});

export default store;
