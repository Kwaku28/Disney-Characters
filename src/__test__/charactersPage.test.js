import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import CharactersPage from '../components/pages/charactersPage';
import store from '../redux/store';

it('Renders the exact page', () => {
  const root = renderer.create(
    <Provider store={store}>
      <Router>
        <CharactersPage />
      </Router>
    </Provider>,
  ).toJSON;
  expect(root).toMatchSnapshot();
});
