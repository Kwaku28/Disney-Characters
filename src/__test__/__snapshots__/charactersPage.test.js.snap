// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the exact page 1`] = `[Function]`;
